app/components/address-bar.js

```diff
@@ -0,0 +1,9 @@
+import Ember from "ember";
+
+export default Ember.Component.extend({
+  classNames: ["url-bar"],
+
+  change(event) {
+    this.sendAction("urlChanged", event.target.value);
+  }
```

Comment:
Just to be sure, this will be fired when a new URL is entered and the element is blurred, right? In the previous commit, the action is only fired, when enter is pressed...